### PR TITLE
Correctly work with 404 error from OBS in approver

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: openqabot/approver.py [hunk 1]
```diff
@@ -123,11 +123,17 @@ def osc_approve(inc: IncReq) -> bool:
             )
         except HTTPError as e:
             if e.code == 403:
-                logger.debug(
+                logger.error(
```
Comment (okurz) [inline openqabot/approver.py:126]: If qem-bot already takes the action of ignoring, why log as error?

**Review by okurz — CHANGES_REQUESTED** [review]

**Review by perlpunk — COMMENTED** [review]
I think I would find it helpful to also log the url.

**Comment by mimi1vx** [comment]
> I think I would find it helpful to also log the url.

URL will be pretty awful (obs api) , what will be helpful is Request number

File: openqabot/approver.py [hunk 2]
```diff
@@ -123,13 +123,22 @@ def osc_approve(inc: IncReq) -> bool:
...
         except HTTPError as e:
             if e.code == 403:
-                logger.debug(
+                logger.info(
                     "Received '%s'. Request likely already approved, ignoring"
                     % e.reason
                 )
                 return True
+            elif e.code == 404:
+                logger.error(
```
Comment (okurz) [inline openqabot/approver.py:132]: still here I would use `logger.debug` (or `logger.info`) as there is already the action mentioned that is conducted when the situation is hit. We should ask ourselves what a reader of an error log message should do. If it's nothing that needs to be done then we also don't need to log with that high level

**Review by okurz — CHANGES_REQUESTED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**